__tests__/database.test.ts

```diff
@@ -89,10 +89,10 @@ describe('Database', () => {
         await deleteSession(session.token)
     })
 
-    test('can delete a user', () => deleteUser('jdoe@murraystate.edu')
-        .then(_ => true)
-        .catch(_ => false)
-    )
+    test('can delete a user', async () => {
+        await deleteUser(testUser.email)
+        return true
```

Comment:
Won't this automatically return true if no errors occur?

Reply:
Yes. It might be better to add a check after the delete to see if the user still exists in the database.